src/plugins/auto/plugin.cpp

```diff
@@ -192,25 +192,32 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::ParseMetaDevices(cons
         for (auto&& deviceNameWithID : sameTypeDevices) {
             DeviceIDParser newParsed{deviceNameWithID};
             std::string defaultDeviceID = "";
+            std::string tempDeviceID = "";
             if (newParsed.getDeviceID().empty()) {
                 defaultDeviceID = getDefaultDeviceID(deviceNameWithID);
+                tempDeviceID = defaultDeviceID;
```

Comment:
what is the difference between `tempDeviceID` and `defaultDeviceID` ?

Reply:
this change is to make sure that the metaDevice contain deviceID only in cases like -d MULTI:GPU. when multiple devices like -d MULTI:GPU.0,GPU.1, the device id will not be parsed into metaDevice struct. I'm fixing the issue in this way to:
1) fix the logic and make sure below code able to find remote tensor affinity correctly:
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/multi_schedule.cpp#L74
2) meanwhile this change will have minimum impact on previous test cases